R-package/src/cmake/modules/FindLibR.cmake

```diff
@@ -10,7 +10,6 @@
 #  LIBR_HOME
 #  LIBR_EXECUTABLE
 #  LIBR_INCLUDE_DIRS
-#  LIBR_LIB_DIR
 #  LIBR_CORE_LIBRARY
 # and a CMake function to create R.lib for MSVC
```

Comment:
I think `R_DOT_LIB_FILE` should be listed here and included in `find_package_handle_standard_args` conditionally.
Also just found the following guide: https://cmake.org/cmake/help/v3.0/manual/cmake-developer.7.html?highlight=find_package_handle_standard_args#find-modules.

Reply:
I liked `R_DOT_LIB_FILE` because it was so different from `LIBR_CORE_LIBRARY` that they were unlikely to be mistaken for each other.

However, I just changed it to `R_MSVC_CORE_LIBRARY`. I don't think the name is that important and I'm anxious to get this merged. MSVC installation for the R package is currently broken for users (https://github.com/microsoft/LightGBM/pull/2963#issuecomment-619646274) and I believe it has been for a few weeks now.

PR with the changes: https://github.com/microsoft/LightGBM/pull/2963/commits/c283c55ca5f3a9bbd832634046866c721dcc4849